### PR TITLE
Parse maps

### PR DESCRIPTION
**Pull request by CurryNRice**
Can now get infortmation about gIds and multiple layers.
Co-authored-by: Supergamer1337

### CODE REVIEW

File: desktop/src/com/g16/feyrune/DesktopLauncher.java [hunk 1]
```diff
@@ -3,6 +3,8 @@
 import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application;
 import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration;
 import com.g16.feyrune.Feyrune;
+import com.g16.feyrune.map.Map;
+import com.g16.feyrune.model.Parser.MapParser;
```
Comment (Supergamer1337) [inline desktop/src/com/g16/feyrune/DesktopLauncher.java:7]: Should this still be here? Is it used?

File: core/src/com/g16/feyrune/map/Map.java [hunk 2]
```diff
@@ -9,7 +9,7 @@ public class Map {
     private final int width;
     private final int height;
 
-    protected Map(Tile[][] tiles) {
+    public Map(Tile[][] tiles) {
```
Comment (Supergamer1337) [inline core/src/com/g16/feyrune/map/Map.java:12]: Was it impossible to have a less open "privacy setting" (don't remember the correct term)

Comment (CurryNRice) [reply core/src/com/g16/feyrune/map/Map.java]: You are correct, didnt move everything to the right position

**Review by Supergamer1337 — APPROVED** [review]
Looks good! 👍

**Pull request merged**